application/src/main/resources/thingsboard.yml

```diff
@@ -546,7 +546,7 @@ transport:
     report_timeout: "${TB_TRANSPORT_SESSIONS_REPORT_TIMEOUT:30000}"
   json:
     # Cast String data types to Numeric if possible when processing Telemetry/Attributes JSON
-    type_cast_enabled: "${JSON_TYPE_CAST_ENABLED:true}"
+    type_cast_enabled: "${JSON_TYPE_CAST_ENABLED:false}"
```

Comment:
Please remove this from the fix. This should be set on the environment, not in the main branch.